src/sentry/tasks/post_process.py

```diff
@@ -254,6 +254,9 @@ def post_process_group(
         # renormalize when loading old data from the database.
         event.data = EventDict(event.data, skip_renormalization=True)
 
+        with metrics.timer("tasks.post_process.delete_event_cache"):
+            event_processing_store.delete_by_key(cache_key)
```

Comment:
Do you still need the delete_by_key on line 277?

Reply:
If we delete before all of post_process is complete will we lose the ability to restart post_process on events that may have had post_process fail because of a problem?

Reply:
AFAIK we don't have any logic for restarting/retrying `post_process_group` right now, so I think in that regard we'll be in the same situation.